lib/sandboxed_renderer/init.js

```diff
@@ -43,15 +47,16 @@ preloadProcess.hang = () => binding.hang()
 preloadProcess.getProcessMemoryInfo = () => binding.getProcessMemoryInfo()
 preloadProcess.getSystemMemoryInfo = () => binding.getSystemMemoryInfo()
 preloadProcess.argv = binding.getArgv()
-process.platform = preloadProcess.platform = electron.remote.process.platform
-process.execPath = preloadProcess.execPath = electron.remote.process.execPath
-process.env = preloadProcess.env = electron.remote.process.env
+preloadProcess.platform = process.platform = platform
+preloadProcess.execPath = process.execPath = execPath
+preloadProcess.env = process.env = env
+
 process.on('exit', () => preloadProcess.emit('exit'))
 
 // This is the `require` function that will be visible to the preload script
 function preloadRequire (module) {
   if (preloadModules.has(module)) {
-    return preloadModules.get(module)
+    return require(module)
```

Comment:
Does this work with browserify?

Reply:
What is confusing to me is that we have `fs.js`, `os.js`, `path.js` in `lib/sandboxed_renderer/api/exports`, which are also listed in electron.gyp, but there is no `timers.js` or `url.js`

Reply:
I see, browserify has bundled support for those

Reply:
According to the docs
> Browserify parses the AST for require() calls to traverse the entire dependency graph of your project.

doesn't that mean that all the require calls have to have the module name as literal string?